styles.css

```diff
@@ -0,0 +1,554 @@
+  text-align: center;
+}
+
+hr {
+  width: 327px;
+  height: 0;
+  border: 1px solid #212121;
+  margin-bottom: 4rem;
+}
+
+.title {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+}
+
+.flex h1 {
+  margin-bottom: 1.5rem;
+}
+
+.post1 {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+  border-radius: nullpx;
+}
+
+.text {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  align-items: center;
+  text-align: left;
+  color: #091e42;
+  font-family: 'Inter', serif;
+  width: 327px;
+  height: 147px;
+}
+
+.languages {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 12px;
+  flex-direction: row;
+  width: 327px;
+}
+
+.css {
+  height: 40px;
+  width: 49px;
+  padding: 10px 12px 10px 12px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.html {
+  height: 40px;
+  width: 56px;
+  left: 61px;
+  padding: 10px 12px 10px 12px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.bootstrap {
+  height: 40px;
+  width: 95px;
+  left: 129px;
+  padding: 10px 12px 10px 12px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+}
+
+.Ruby {
+  height: 40px;
+  width: 61px;
+  left: 236px;
+  padding: 10px 12px 10px 12px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+}
+
+.printing-data {
+  background-image: url("images/printing.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 327px;
+  height: 356px;
+  left: 24px;
+  margin: 10px;
+  top: 410px;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 327px;
+  justify-content: center;
+  align-items: space-around;
+}
+
+.printing-title h1 {
+  color: #fff;
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  text-align: left;
+}
+
+.right-block {
+  width: 327px;
+  height: 338px;
+  top: 76px;
+  padding: 8px 16px 8px 16px;
+}
+
+.recent-works {
+  height: 3455px;
+  width: 375px;
+  top: 682px;
+  padding: 120px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 80px;
```

Comment:
- [ ] kindly consider using `display: grid` in this container as this is a project requirement.
- [ ] Also the padding on figma on both sides are `24px` but you are taking `120px` here. so kindly consider fixing it as we need to stick to the figma design.